vizro-core/examples/features/app.py

```diff
@@ -46,8 +50,15 @@
 
                 ### Controls
 
-                Vizro has two different control types **filters** and **parameters**.
+                Vizro has two different control types **Filter** and **Parameter**.
 
+                You can use any pre-existing selector inside the **Filter** or **Parameter**:
+
+                * Dropdown
```

Comment:
Let's remember to add the date picker here when it's ready.

Reply:
@nadijagraca